packages/lego-bricks/src/components/MeterBar/README.md

```
@@ -0,0 +1,33 @@
+<Bar
+  labels={[0,10,20,30,40,50,60,70,80,90,100]}
+  labelColor="steelblue"
+  progress={80}
+  barColor="#fff34b"
+  seperatorColor="hotpink"
+/>
+```
+
+##### Example #2
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)